net-p2p/classified-ads/classified-ads-0.10.ebuild

```diff
@@ -0,0 +1,93 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+PLOCALES="en fi sv da uk"
+PLOCALE_BACKUP="en"
+inherit qmake-utils virtualx vcs-snapshot
+
+COMMIT_ID="cd0652c52f86f6284b793f26e5362bc8fb8a7118"
+DESCRIPTION="Program for displaying classified advertisement items"
+HOMEPAGE="http://katiska.org/classified-ads/"
+SRC_URI="https://github.com/operatornormal/classified-ads/archive/${COMMIT_ID}.tar.gz -> ${P}.tar.gz \
+	https://github.com/operatornormal/classified-ads/blob/graphics/preprocessed.tar.gz?raw=true \
+		-> classified-ads-graphics-${PV}.tar.gz"
+
+LICENSE="LGPL-2.1"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+IUSE="debug doc test"
+
+RDEPEND="dev-libs/openssl:0
+		>=net-libs/libnatpmp-20130911
+		<=net-libs/libnatpmp-20140401-r1
+		>=net-libs/miniupnpc-1.8
+		sys-apps/file
+		sys-devel/gettext
```

Comment:
gettext is usually not required in RDEPEND but only in DEPEND. Please check carefully if it really needs to be in RDEPEND.